packaging/dependency/lxd.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package dependency
+
+import (
+	"github.com/juju/errors"
+	"github.com/juju/juju/packaging"
+)
+
+const blankSeries = ""
+
+// LXD is a dependency instance for installing lxd.
+var LXD packaging.Dependency = lxdDependency{}
```

Comment:
Is it better to have these as declared, or would it be better to have this as a function like we do for Mongo? I feel like it would be good to be consistent.

Reply:
No objection there. I will change this to match the others.